src/components/Person/Person.jsx

```diff
@@ -1 +1,29 @@
-// export const Person = ({ person }) => ();
+export const Person = ({ person }) => {
+  const {
+    name,
+    age,
+    partnerName,
+    sex,
+    isMarried,
+  } = person;
+
+  return (
+    <section className="Person">
+      <h2 className="Person__name">
+        {`My name is ${name}`}
+      </h2>
+      {age && (
+        <p className="Person__age">
+          {`I am ${age}`}
+        </p>
+      )}
+      <p className="Person__partner">
+        {isMarried
+          ? `${partnerName} is my ${
+            sex === 'm' ? 'wife' : 'husband'
```

Comment:
double ternary operator in UI (jsx) is not good
will be better sex checking move to separate variable like this:
`const partnerSexLabel = sex === 'm' ? 'wife' : 'husband';`